src/lib/UIKitConfigProvider/index.tsx

```diff
@@ -36,7 +36,7 @@ const UIKitConfigProvider = ({ storage, children, localConfigs }: UIKitConfigPro
     try {
       if (storage == null) {
         const payload = await sdk.getUIKitConfiguration();
-        setRemoteConfigs(snakeToCamel(payload.json));
+        setRemoteConfigs(snakeToCamel(payload.json).configuration.uikitConfigurations);
```

Comment:
I shouldn't have removed `uikitConfigurations` this one but I guess it's removed by mistake 😅

Reply:
Also, the json data format has been changed in the latest version of sendbird/chat; `configuration` is added in the root level of payload.